packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -137,6 +137,12 @@ const ModalContent = forwardRef((props, ref) => {
         role={role}
         ref={ref}
       >
+        {
+          /* UXPLATFORM-8733: Adding an empty span with data-terra-abstract-modal-begin attribute
```

Comment:
I think we can remove the internal identifier here

Reply:
Since we have not concretely landed on not including JIRA's, we decided to include it as it could be useful for someone internal in triaging especially because we are introducing a new element in the component DOM.

Reply:
Gotcha, I'm fine with that if others agree thats ok since it could be helpful. The reason I was thinking we shouldn't add this is because it wouldn't follow existing Terra documentation given I haven't found Jira numbers in the code anywhere. If anything, the comment is very descriptive so the reason is clear enough and git blame/history should be able to link anyone needing to audit/review the reason can trace it back to this PR which can include the Jira number. @artpark/@sdadn would having the Jira number be ok or is this something that is avoided in the source code?

Reply:
Given that the Jira number is specified in the PR description, would that not give sufficient context as to when/why the change was made?

Reply:
The JIRA is already included in the PR, which can easily be found using the git blame history. I don't think there's much value in adding the JIRA number here. A comment is more than sufficient to convey this information.

Reply:
I was thinking more about when all these changes are squashed and merged into main and if we need to review a year from now. That said, we are good with removing the JIRA reference.